docs(gitlabrunner): add GitLab registration details ext-fix-gitlabrunner (#3960)
scaleway/docs-content: tutorials/easydeploy-gitlab-runner/index.mdx

        If you cannot find GitLab Runner on the first page, use the search bar or navigate through the library using the arrow buttons.
    </Message>
7. Optionally, customize the default configuration for GitLab Runner using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not need any customized configuration you can skip this step.

> 8. **Provide GitLab Details:**

Reviewer: ```suggestion
8. **Provide the following GitLab details:**
```

>     - **GitLab URL (`gitlabUrl`)**: Go to your GitLab project, navigate to **Settings > CI/CD > Runners**, and copy the GitLab instance URL.

Reviewer: ```suggestion
    - **GitLab URL (`gitlabUrl`)**: Go to your GitLab project, click **Settings > CI/CD > Runners**, and copy the GitLab instance URL.
```

    - **Registration Token (`runnerToken`)**: Under the **Specific Runners** section in **Settings > CI/CD > Runners**, copy the registration token for your project. These are necessary to allow GitLab Runner to register correctly with your GitLab project.
    ```yaml
    gitlabUrl: https://your-gitlab.com
    runnerToken: "your-token"
    rbac:
      create: true
    ```

>     <Message type="info">

Reviewer: ```suggestion
    <Message type="note">
```

>         It is essential to provide these details to successfully register your GitLab Runner with your GitLab project.

Reviewer: ```suggestion
        The information above is required to register your GitLab Runner with your GitLab project.
```

    </Message>

> 9. Enter a name (e.g. `gitlab-runner`) and a Kubernetes namespace for your application. If no name is entered, GitLab Runner will be installed in the default namespace of the cluster.

Reviewer: ```suggestion
9. Enter a name (e.g. `gitlab-runner`) and a Kubernetes namespace for your application. If you do not specify a name, GitLab Runner will be installed in the default namespace of the cluster.
```

10. Click **Deploy Application** to deploy GitLab Runner on your Kubernetes cluster.

> 11. Verify that the GitLab Runner is successfully installed and running:

Reviewer: ```suggestion
11. Use the following command to verify that the GitLab Runner is installed and running:
```
